README.md

```diff
@@ -1,53 +1,134 @@
 [![Automatic version updates](https://github.com/ZOSOpenTools/v8port/actions/workflows/bump.yml/badge.svg)](https://github.com/ZOSOpenTools/v8port/actions/workflows/bump.yml)
 
-v8
+# v8
 
 A port of Google V8 JavaScript engine to z/OS Open Tools project
 
-# Applying the patches
+## Workflow 
+
+### Applying the patches
 Since the [cipd executable](https://chromium.googlesource.com/infra/luci/luci-go/+/master/cipd/) is not yet ported on z/OS, the following steps are
 currently used to update V8 and its dependencies, and apply the z/OS port, after
 the initial [fetch v8](https://v8.dev/docs/source-code) (also included below with a comment).
 
+This workflow requires two platforms.  One a z/OS® UNIX System Services (z/OS UNIX) which is 
+the target platform.  The second is a generic platform used to get files normally acquired via `cipd`.
+
+
+#### Linux plaform (non z/OS® UNIX System Services (z/OS UNIX)) setup
+
+This is the workflow on the non-z/OS UNIX platform for obtaining files via `cipd`.  
+There are two aspects to this workflow.  One which is done once and one which
+is done everytime you want to sync.
+
+This workflow is done once.
+
 ```
-# Do this only once:
-mkdir ~/v8base
-cd v8base
-fetch v8
-cd v8
+$ cd $HOME
+$ git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+$ export PATH=~/depot_tools:$PATH
```

Comment:
We still need to use depot_toolsport to use its fetch and gclient commands. Please remove those steps and instead add at the very beginning a comment like "Setup your depot_tools environment as instructed in https://github.com/ZOSOpenTools/depot_toolsport.".